libraries/tools/kotlin-stdlib-gen/src/templates/Aggregates.kt

```diff
@@ -65,7 +65,7 @@ object Aggregates : TemplateGroupBase() {
             Returns `true` if no ${f.element.pluralize()} match the given [predicate].
             """
         }
-        sample("samples.collections.Collections.Aggregates.noneWithPredicate")
+        sample("samples.text.Strings.Aggregates.noneWithPredicate")
```

Comment:
This change updates sample for all collections, maps and arrays specified in function declaration above (`includeDefault(); include(Maps, CharSequences, ArraysOfUnsigned)`).

To use a new sample for char sequences only, you need to leave that line as it was before and add the following right after it: 
```kotlin
specialFor(CharSequences) {
    sample("<your new sample>")
}
```